web/templates/account/edit_account.html

```diff
@@ -1,184 +1,363 @@
+                                                    y="7" width="16" height="2" rx="1"></rect>
+                                                <rect
+                                                    class="origin-center group-[[aria-expanded=true]]:rotate-[135deg] group-[[aria-expanded=true]]:[y:7] group-[[aria-expanded=true]]:[x:0] transition-all duration-300 ease-[cubic-bezier(.5,.85,.25,1.1)]"
+                                                    y="12" width="9" height="2" rx="1"></rect>
+                                            </svg>
+                                        </button>                                                
+                                        <!-- Breadcrumbs -->
+                                        <div class="flex items-center text-sm whitespace-nowrap min-w-0 ml-3">
+                                            <span class="text-gray-600">Dashboard</span>
+                                            <svg class="fill-gray-400 shrink-0 !mx-2 -rotate-90" width="11" height="7" xmlns="http://www.w3.org/2000/svg">
+                                                <path d="m2 .94 3.5 3.5L9 .94 10.06 2 5.5 6.56.94 2 2 .94Z" />
+                                            </svg>
+                                            <span class="font-medium truncate">Usage</span>
+                                        </div>
+                                    </div>                                        
+
+                                    <!-- Article content -->
+                                    <div>
+                                        <header class="mb-10">
+                                            <h1 class="text-3xl font-bold mb-3">Account Dashboard</h1>
+                                            <p class="text-lg text-gray-700">You are on {{ request.user.get_tier_display|lower }} plan</p>
+                                        </header>
+                                        <div class="prose text-gray-700 max-w-none prose-headings:font-bold prose-headings:text-gray-900 prose-a:font-medium prose-a:text-blue-500 prose-a:no-underline hover:prose-a:underline prose-strong:font-medium prose-strong:text-gray-900 prose-code:font-mono prose-code:text-gray-900 prose-code:bg-gray-100 prose-code:px-1 prose-code:py-0.5 prose-code:rounded prose-code:before:content-[''] prose-code:after:content-[''] prose-pre:bg-gray-900 prose-pre:border prose-pre:border-gray-700 prose-headings:scroll-mt-24">
+                                            <!-- The data-scrollspy-target attribute makes the scrollspy work -->
+                                            <h2 data-scrollspy-target id="usage">API Usage</h2>
+                                            <p>
+                                                {{ credits_used }} credits used out of {{ request.user.available_credits }} credits in the past 30 days.
```

Comment:
this wouldn't work for non-free users, since their available credits wouldn't include the 2500/25000 monthly credits. Available credits basically only includes the 1000 free signup credits and all other credits are tracked by stripe

Reply:
We will need to query stripe info for paid users anyway so I think it will look something like this:
- Free users: "You have {available_credit} remaining"
- Paid users: "You have {2500 - stripe usage} credits remaining, additional usage will be billed at 0.01 $ per credit" 
- Unless their usage exceeds their credits then it would say "You have used all your credits, upgrade to a paid plan to get more credits" or "You have used all your monthly credits, additional usage will be billed at 0.01 $ per credit"